src/pages/login.tsx

```diff
@@ -0,0 +1,200 @@
+import {
+  Box,
+  Button,
+  Checkbox,
+  Flex,
+  Image,
+  Input,
+  Link,
+  Text,
+  FormControl,
+  FormErrorMessage,
+  InputGroup,
+  InputRightElement,
+} from "@chakra-ui/react";
+import { NavLink } from "react-router-dom";
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+
+type InputValues = {
+  email: string;
+  password: string;
+  name: string;
+};
+
+const Login = () => {
+  const Logo = "assets/WC-LOGO-july.png";
+  const initialValues: InputValues = {
+    name: "",
+    email: "",
+    password: "",
+  };
+
+  const [input, setInput] = useState(initialValues);
+  const [show, setShow] = useState(false);
+
+  const handleClick = () => setShow(!show);
+
+  const handleInputChange = (e: any) =>
+    setInput({
+      ...input,
+      [e.target.name]: e.target.value,
+    });
+
+  const isError = input.password === "";
+
+  const navigate = useNavigate();
+
+  const redirectToHome = (e: React.FormEvent<HTMLButtonElement>) => {
+    e.preventDefault();
+    navigate("/admin/dashboard", { replace: true });
+  };
+
+  return (
+    <Box>
+      <Box
+        bg="#021D37"
+        h="200px"
+        w="100%"
+        zIndex="-3"
+        position="relative"
+      ></Box>
+
+      <Box
+        flexDirection="column"
+        alignItems="center"
+        mt="-80px"
+        borderRadius="5px"
+        p="30px"
+        w={{
```

Comment:
Can you create a dumb component called "SignUpLayout" abstract the UI jsx that is contained in this file

Reply:
I'd be doing the same for the login right ?